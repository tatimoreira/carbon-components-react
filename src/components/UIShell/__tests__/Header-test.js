import React from 'react';
import { mount } from 'enzyme';
import { Header } from '../';

describe('Header', () => {
  it('should render', () => {
    const wrapper = mount(<Header className="custom-class" />);
    expect(wrapper).toMatchSnapshot();
  });
});
