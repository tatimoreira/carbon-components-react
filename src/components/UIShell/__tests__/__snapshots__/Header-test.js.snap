// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header should render 1`] = `
<Header
  className="custom-class"
>
  <header
    className="bx--platform-header custom-class"
  />
</Header>
`;
